crates/biome_js_analyze/tests/specs/nursery/noFlatMapIdentity/valid.js.snap

```diff
@@ -0,0 +1,8 @@
+---
+source: crates/biome_js_analyze/tests/spec_tests.rs
+expression: valid.js
+---
+# Input
+```jsx
+array.flatMap((arr) => arr * 5);
```

Comment:
The `valid.js` is `arr * 2` but here it's `arr * 5`, something wrong here I think.

Reply:
my bad, i just change valid.js to make sure the documentation same, do i need to do test and genlint again, or just change that number 5 to 2 in snap file?

Reply:
It should be same. An easy way is delete the valid.js snap file and recreate it by just test command.

Reply:
done